src/GitVersionCore.Tests/VariableProviderTests.cs

```diff
@@ -152,4 +152,29 @@ public void ProvidesVariablesInContinuousDeploymentModeForStableWhenCurrentCommi
 
         JsonOutputFormatter.ToJson(vars).ShouldMatchApproved(c => c.SubFolder("Approved"));
     }
+
+    [Test]
+    [Category("NoMono")]
+    [Description("Won't run on Mono due to source information not being available for ShouldMatchApproved.")]
```

Comment:
The first failure I got seemed to be related to the tests running on mono, but you are right it looks like 228082b did pass the build.